projects/FinalProject-Bugs/dominion/unittest10.c

```diff
@@ -0,0 +1,103 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "dominion.h"
+#include <string.h>
+#include "rngs.h"
+#include "dominion_helpers.h"
+
+int main()
+{
+	int i, r;
+	int currentPlayer = 0;
+	int discardCount;
+	int handCount;
+	int k[10] = { ambassador, council_room, feast, gardens, mine, remodel, smithy, village, treasure_map, great_hall };
+	int maps;
+	int nextPlayer = 1;
+	int numPlayers = 2;
+	int seed = 1000;
```

Comment:
We probably should just use a random number for seed.

Reply:
That's a good idea. I suppose this should be changed in @gibsojen 's tests too?

Reply:
Good idea.  I will update mine too